pipeline/01_parser.py

```
@@ -213,31 +250,45 @@ def parse_docx(file_path):
+                text = parse_hwp(inner_path)
+            elif ext in [".xlsx", ".xls"]:
+                text = parse_xlsx(inner_path)
+            elif ext == ".docx":
+                text = parse_docx(inner_path)
+
+            if text:
+                # ZIP 안에 같은 이름이 다른 디렉터리에 있을 수 있어 상대 경로 사용
+                rel = os.path.relpath(inner_path, tmp_zip_dir)
+                result.append(f"[ZIP 내부: {rel}]\n{text}")
 
     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)